test/test-clone.sh

```diff
@@ -144,12 +144,14 @@ begin_test "clone ClientCert"
   set -e
   reponame="test-cloneClientCert"
   setup_remote_repo "$reponame"
+  pwd
   out="$(clone_repo_clientcert "$reponame" "$reponame")"
```

Comment:
> I can't use `tee` exactly, since it swallows errors.

You should still be able to capture non-zero exit codes with `${PIPESTATUS[0]}`:

```sh
thing_that_might_fail 2>&1 | tee foo.log
ok="${PIPESTATUS[0]}"
```

But I don't mind either way which approach you take here 👍